Connectors/Rhino/Speckle.Connectors.RhinoShared/Bindings/RhinoSendBinding.cs

```diff
@@ -316,34 +343,38 @@ private async Task RunExpirationChecks()
       }
     }
 
-    if (ChangedObjectIds.IsEmpty && ChangedObjectIdsInGroups.IsEmpty)
+    if (ChangedObjectIds.IsEmpty && ChangedObjectIdsInGroupsOrLayers.IsEmpty)
     {
       return;
     }
 
     // Actual model card invalidation
     string[] objectIdsList = ChangedObjectIds.Keys.ToArray();
-    var changedObjectIdsInGroups = ChangedObjectIdsInGroups.Keys.ToArray();
+    var changedObjectIdsInGroups = ChangedObjectIdsInGroupsOrLayers.Keys.ToArray();
     _sendConversionCache.EvictObjects(objectIdsList);
     var senders = _store.GetSenders();
     List<string> expiredSenderIds = new();
 
     foreach (SenderModelCard modelCard in senders)
     {
       var intersection = modelCard.SendFilter.NotNull().SelectedObjectIds.Intersect(objectIdsList);
-      var groupIdIntersection = modelCard.SendFilter.NotNull().SelectedObjectIds.Intersect(changedObjectIdsInGroups);
-
-      var isExpired = intersection.Any() || groupIdIntersection.Any();
+      if (intersection.Any())
+      {
+        expiredSenderIds.Add(modelCard.ModelCardId.NotNull());
+        continue;
+      }
 
-      if (isExpired)
+      var groupIdIntersection = modelCard.SendFilter.NotNull().SelectedObjectIds.Intersect(changedObjectIdsInGroups);
```

Comment:
same for `groupIdIntersection` -> `groupOrLayerIdIntersection`

Reply:
done!